job_templates/ci_job.xml.em

```diff
@@ -207,7 +207,19 @@ echo "# BEGIN SECTION: Build Dockerfile"
 @[    if os_name == 'linux-aarch64']@
 docker build --pull ${DOCKER_BUILD_ARGS} --build-arg PLATFORM=aarch64 -t ros2_batch_ci_aarch64 linux_docker_resources
 @[    elif os_name == 'linux-rhel']@
-docker build --pull ${DOCKER_BUILD_ARGS} -t ros2_batch_ci_rhel linux_docker_resources -f linux_docker_resources/Dockerfile-RHEL
+# FIX ME !!
+# Due to issues with pyside2, EPEL and AlmaLinux 9.3 it's impossible to build docker images; to get ci working we reverted to a previous built stale image.
+# This ought to be temporary until pyside2 can be rebuilt to work with AlmaLinux 9.3. See https://github.com/ros2/ci/issues/727. 
+# An if-else scenario is added to be able to quickly test if it has been resolved when the rebuilt happens.
```

Comment:
I'm not sure the conditional is justifiable as the real test will be reverting this change once ready. I could be talked out of it but in general I prefer not to add conditions to these scripts and keep them as sigh-readable as possible.

Reply:
@nuclearsandwich I see this more in terms of the effort necessary to test that the issue with AlmaLinux 9.3 and EPEL has been resolved. If I don't leave the mechanism to test easily in the form of the conditional  while reverting this PR, I would need to redeploy the new changes to the test jobs and do the testing vs simply running ci with a different parameter.  This strategy was what allowed me to test the build regressions across different versions of AlmaLinux without needed to redeploy to test each in the closed PR linked. 
Furthermore since both changes are contained in this PR  the new addition should be removed in the same process of reverting the stale images changes and won't need extra PRs of cleanup. 
I prefer to test that the change has worked before reverting to a different state on prod.

Reply:
When there isn't an extraordinary concern, implementor's preference has precedence.